tests/rule_based_profiler/data_assistant/test_column_value_missing_data_assistant.py

```diff
@@ -27,3 +32,45 @@ def test_column_value_missing_data_assistant_result_plot_expectations_and_metric
     # This test passes only if absense of any metrics and expectations to plot does not cause exceptions to be raised.
     column_domain_charts: List[dict] = [p.to_dict() for p in plot_result.charts[2:]]
     assert len(column_domain_charts) == 0
+
+
+@pytest.mark.integration
+@pytest.mark.slow
+def test_single_batch_multiple_columns(empty_data_context):
```

Comment:
what type of context is this? I want to say file - I think we can leverage an ephemeral context so we don't have side effects here

Reply:
@cdkini Our `empty_data_context` is indeed `FileDataContext` with nothing in it.  @TrangPham please feel free to try `ephemeral_context_with_defaults` and see if it works.  (It is defined in "tests/conftest.py".) Thank you both.